src/PositionManager.sol

```diff
@@ -128,69 +128,75 @@ contract PositionManager is
     }
 
     function _handleAction(uint256 action, bytes calldata params) internal virtual override {
-        if (action == Actions.INCREASE_LIQUIDITY) {
```

Comment:
this looks bad but i just split the command into a nested if for gas. If you view it in codespace itll make it much easier to see what i did lol.
its like: 
```
if (action > settle) {
   ... all liquidity / mint / burn commands ...
} else {
   ... all payment commands ...
}
```